modules/matterhorn-index-service/src/main/java/org/opencastproject/index/service/catalog/adapter/events/CommonEventCatalogUIAdapter.java

```diff
@@ -77,8 +81,24 @@ public Catalog storeFields(MediaPackage mediaPackage, MetadataCollection abstrac
     }
 
     MetadataField<?> series = abstractMetadata.getOutputFields().get(DublinCore.PROPERTY_IS_PART_OF.getLocalName());
-    if (series != null && series.isUpdated() && isNotBlank(series.getValue().get().toString()))
-      mediaPackage.setSeries(series.getValue().get().toString());
+    if (series != null && series.isUpdated() && isNotBlank(series.getValue().get().toString())) {
```

Comment:
annoyingly isUpdated() resturns false if series is set  to no option as the logic should be

Reply:
and then it updates the event from the episode catalog that has not been updated with the fact it is not longer partOf anything...

I do not have the time to rewrite OC to properly handle unsetting values, when all this was meant to do was handle changes in series.  Can we either accept the PR as is and live with the fact it doesn't handle removing an event from  a series or simply reject it? I assume this has all be fixed upstream.

Reply:
@JamesUoM I haven't looked at your pull, and I'm not sure this will help, the purpose of https://github.com/opencast/opencast/pull/56/files is the ability to marshall intentionally empty fields in an DC catalog for the purpose of being able to merge intentioally empty targeted fields into an existing catalog. In other words, to be able to empty out specific fields in a catalog. If you leave this pull open for a while longer. I can try take a look in the next couple days to see if it, or a companion strategy, can be applied here.

Reply:
@wsmirnow Without that else case, what happens?  We're at the point where either we accept as is or decline it.  Is it better to have this patch as is, or not?  3.6 releases in two days.